.env.example

```diff
@@ -0,0 +1,5 @@
+# The base API endpoints to which requests are made
+REACT_APP_BOOKSTORE_URL=https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps
+
+# check the api documentation at https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739
+REACT_APP_BOOKSTORE_API_KEY=NSzgaZs3adB76DpE8Clb
```

Comment:
- [ ] Good job, but your `REACT_APP_BOOKSTORE_URL` was not available to the rest of your code making all API call undefined till I renamed this file to `.env` so kindly remove the `.`example` you added to the name of this file